eva-lib/src/main/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepository.java

```diff
@@ -60,4 +63,11 @@ Long countByChromosomeAndStartAndReferenceAndAlternate(String chromosome, int st
 
     @Query(value = "{'chr': ?0, 'start': ?1, 'ref': ?2}", count = true)
     Long countByChromosomeAndStartAndReference(String chr, int start, String ref);
+
+    @Query(value = "{'chr': ?0, 'start': ?1, 'alt': ?2, 'files.sid': {$in : ?3}}}", count = true)
+    Long countByChromosomeAndStartAndAltAndStudyIn(String chr, int start, String alt, List<String> studyIds);
+
+    @Query(value = "{'chr': ?0, 'start': ?1, 'type': ?2, 'files.sid': {$in : ?3}}}", count = true)
+    Long countByChromosomeAndStartAndTypeAndStudyIn(String chr, int start, Variant.VariantType type,
+                                                    List<String> studyIds);
```

Comment:
I am wondering about the usefulness of some of these `countBy` methods: for paginated queries they are useful because they allow to calculate the total number, not just for a single page. But for non-paginated results we could just have `findBy` methods that are be more flexible in terms of data that can be obtained. The number of results would be the size of the in-memory collection.

This would mean that:
* `countByChromosomeAndStartAndReference(AndAlternate)` are no longer necessary
* `countByChromosomeAndStartAndAltAndStudyIn` and `countByChromosomeAndStartAndTypeAndStudyIn` are converted into `findBy` methods